Marlin/Marlin_main.cpp

```diff
@@ -2306,7 +2306,7 @@ static void clean_up_after_endstop_or_probe_move() {
    *   - Raise to the BETWEEN height
    * - Return the probed Z position
    */
-  float probe_pt(const float &x, const float &y, const bool stow/*=true*/, const int verbose_level/*=1*/) {
+  float probe_pt(const float &x, const float &y, const bool stow/*=true*/, const int verbose_level/*=1*/, const bool printable = true) {
```

Comment:
Apparently those other args are obsolete. So this can be changed to:
```cpp
  float probe_pt(const float &x, const float &y, const bool stow, const uint8_t verbose_level, const bool printable=true) {
```